### ISSUE TITLE
Saving of models with slug does not work

### ISSUE THREAD

**Issue by o15a3d4l11s2** [cviebrock/eloquent-sluggable#545]
After updating from 8.0.2 to 8.0.3 the models that are sluggable can no longer be saved. Saving fails on `$this->fireModelEvent('saving')`. Is there a breaking change that I could not find in the latest version?

I noticed the change from "saving" to "saved", but as the default behaviour is still untouched, I cannot understand what could be wrong.

Here is a simple definition I use:
```php
use Cviebrock\EloquentSluggable\Sluggable;

// Inside the class
use Sluggable;

public function sluggable()
{
    return [
        'slug' => [
            'source' => 'name'
        ]
    ];
}
```

**Comment by guysolamour** [comment]
Have the same issue

**Comment by cviebrock** [comment]
@guysolamour and @o15a3d4l11s2 Try upgrading to 8.0.4 please.

**Comment by o15a3d4l11s2** [comment]
@cviebrock, I can confirm that 8.0.4 works properly for me.

**Comment by shanecp** [comment]
Can confirm 8.0.3 is broken. Maybe remove that version to stop it going to production?
Spent about 3 hours trying to troubleshoot it. Fixed with 8.0.4

Thanks.

**Comment by cviebrock** [comment]
Done ... thanks.
